### PR TITLE
EN - Fixed Bug: Made "UCSB Courses Search" button appear clearly as button

### PR DESCRIPTION
**Pull request by ericknee**
Fixes Issue [#23](https://github.com/ucsb-cs156-s24/proj-courses-s24-4pm-1/issues/23)  
Made "UCSB Courses Search" link on main page Navbar appear clearly as button  
Edited AppNavbar.js

Before: 
Screenshot 2024-05-27 at 5 32 14 PM

After:
Screenshot 2024-05-27 at 4 55 05 PM

Dokku deployment: https://project-ericknee-dev.dokku-01.cs.ucsb.edu/

### CODE REVIEW

**Comment by pconrad** [comment]
For the Stryker gap:

image

Two possible solutions:
* Factor out the CSS into a section that you can use `// Stryker disable all` and then `// Stryker restore all` to exclude
* Write a test similar to this one from `frontend/src/tests/components/BasicCourseSearch/CourseOverTimeInstructorSearchForm.test.js`

```
    expect(buttonRow).toHaveAttribute(
      "style",
      "padding-top: 10px; padding-bottom: 10px;",
    );
```

**Comment by pconrad** [comment]
I've added the FIXME-Stalled label since this PR doesn't seem to be moving forward.  The team may want to see if the author needs help getting it moving.

**Review by Frank404NotFound — APPROVED** [review]
LGTM

File: frontend/src/index.css [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+}
 
-  .btn-primary, .btn-primary:hover, .btn-primary:focus{
-    background-color: #34859b;
-    border-color: #34859b;
-  } 
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
```
Comment (pconrad) [inline frontend/src/index.css:13]: What is this doing in the PR?  Seems unrelated.

File: frontend/src/index.css [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+  background-color: rgb(0, 54, 96);
+}
 
-  
+.btn-primary,
+.btn-primary:hover,
+.btn-primary:focus {
+  background-color: #34859b;
+  border-color: #34859b;
+}
```
Comment (pconrad) [inline frontend/src/index.css:23]: I see that this is only formatting differences.  But this kind of stuff clutters up a PR and makes it harder to review.

In industry, this is considered a sloppy practice, so I'd encourage you to avoid it.

**Review by pconrad — APPROVED** [review]
I'm approving this in spite of the formatting improvements to unrelated code.

As I pointed out on the slack channel:

While improving formatting is a good thing to do, having files in your PR that are unrelated to the change you are making makes:
* Code review more complicted
* Makes code review take longer.
* May be considered "sloppy practice" by some professional developers.

So, please get in the habit of not doing this.  Instead, make a separate PR for the formatting improvements.  In this class, that extra PR would have earned the team 5 extra points.

**Review by monicatrinh1 — COMMENTED** [review]
LGTM
